src/viam/sdk/rpc/dial.hpp

```diff
@@ -27,7 +27,10 @@ class ViamChannel {
 
 class Credentials {
    public:
+    /// Deprecated use Credentials(std::string type, std::string payload)
```

Comment:
can we change the start of this comment to `@deprecated`? That should tell doxygen to mark the method as deprecated in the docs we generate. 

Also: adding back ticks around the preferred `Credentials` method would be good for proper formatting.